docs/vgclog.md

```diff
@@ -0,0 +1,260 @@
+
+## Verbose GC log contents and structure 
+The verbose GC logs are printed in XML format and consist of the following sections:  
+
+- A summary of your GC configuration, which is captured in the `<initialized>` XML element. 
+
+- Information about the GC cycles that ran, including each cycle's GC operations and GC increments.  
+
+For definitions of GC cycles, operations, phases, see [Garbage collection policies](gc.md#garbage-collection-policies). For definitions of GC increments, see [GC increments and interleaving](#gc-increments-and-interleaving).
+
+The logs record when GC cycles and their increments start and end, and list the GC operations that run within these increments to manage or reclaim memory. You can also determine which type of event triggered the cycle or increment, and the amount of memory available to your application before and after processing.  
+
+Not all operations that take place during GC are recorded. For example, the logs record when an increment of a concurrent GC cycle starts and ends, but most concurrent GC operations are not recorded.
+
+### Initialization
+
+The first section of the log records the configuration of the garbage collector, for example:  
+
+```xml
+<initialized id="1" timestamp="2020-10-18T13:27:07.691"> 
+
+    <attribute name="gcPolicy" value="-Xgcpolicy:gencon" /> 
+
+    <attribute name="maxHeapSize" value="0x40000000" /> 
+
+    <attribute name="initialHeapSize" value="0x40000000" /> 
+
+    <attribute name="compressedRefs" value="true" /> 
+
+... 
+ 
+<system> 
+
+    <attribute name="physicalMemory" value="100335456256" /> 
+
+    <attribute name="numCPUs" value="28" /> 
+
+... 
+
+</system> 
+
+<vmargs> 
+
+    <vmarg name="-Xoptionsfile=/java/perffarm/sdks/O11_j9_x64_linux-20201014/sdk/lib/options.default" /> 
+… 
+    <vmarg name="-Dsun.java.launcher.pid=6068" /> 
+
+</vmargs> 
+
+</initialized> 
+
+</verbosegc> 
+
+UNNAMED" /> 
+
+<vmarg name="--add-opens=java.sql/java.sql=ALL-UNNAMED" /> 
+
+<vmarg name="--add-opens=java.management/javax.management=ALL-UNNAMED" /> 
+... 
+
+</vmargs> 
+
+</initialized> 
+```
+
+ADD DETAILS ABOUT THE INITIATISATION SECTION
+
+The first set of <attribute> XML elements record the GC policy type and its settings. 
+The child <attribute> XML elements of the <system> stanza records information about the....
+<region> XML elements...
+<vmargs>....
+
+The verbose GC log then begins recording GC activities and details. 
+
+### GC Cycles
+
+The start of a GC cycle is recorded by the `<cycle-start>` XML element. The trigger for the GC cycle might be captured in a preceding element, for example if the trigger is an allocation failure. In other cases, the trigger might be captured in a subsequent element, for example if a policy's memory threshold trigger has been reached.
+
```

Comment:
Not sure what you meant by 'subsequent element' . Trigger is reported always before cycle-start element. You can simplify and state that most common triggers are allocation failure and allocation taxation, 

There might be some more complex triggers, like percolating GC from scavenge and global, which we did not discuss yet, and probably should not be mentioned here either. 

Also triggers could be reported not only for cycle start, but for a increment starts as well.

Reply:
Rewritten, ready for another review.